lib/takeaway.rb

```diff
@@ -0,0 +1,77 @@
+require_relative 'order'
+require 'twilio-ruby'
+
+
+
+class Takeaway
+
+  attr_reader :menu, :basket, :order_cost
+
+  def initialize(menu)
+    @menu = menu
+    @basket = {}
+    @order_cost = Order.new
+  end
+
+  def show_menu
+    @menu.dishes.dup
+  end
+
+  def order(item, number=1)
+    @basket["#{item} x#{number}"] = @menu.price(item) * number
+    @order_cost.add(price(item,number))
+    "You ordered #{item} x#{number}"
+  end
+
+  def total
+    "Total order cost: £#{@order_cost.total}"
+  end
+
+  def check_total
+    @order_cost.total == basket_summary
+  end
+
+  def reset
+    @basket = {}
+    @order_cost = 0
+  end
+
+  def complete_order
+    send_text(message)
+    reset
+    "Thank you, come again."
+  end
+
+  def basket_summary
+    _basket_total = 0
```

Comment:
Lint/UnderscorePrefixedVariableName: Do not use prefix `_` for a variable that is used.